mcs/build/tests.make

```diff
@@ -21,8 +21,9 @@ ifndef NO_TEST
 
 test_nunit_lib = nunitlite.dll
 xunit_core := xunit.core xunit.execution.desktop xunit.abstractions xunit.assert Xunit.NetCore.Extensions
-xunit_deps := System.Runtime
-xunit_src  := $(patsubst %,$(topdir)/../external/xunit-binaries/%,BenchmarkAttribute.cs BenchmarkDiscover.cs) $(topdir)/../mcs/class/test-helpers/PlatformDetection.cs
+xunit_deps := System.Runtime System.Reflection System.Threading.Tasks
+xunit_src  := $(patsubst %,$(topdir)/../external/xunit-binaries/%,BenchmarkAttribute.cs BenchmarkDiscover.cs) $(topdir)/../mcs/class/test-helpers/PlatformDetection.cs \
+$(topdir)/../external/corefx/src/CoreFx.Private.TestUtilities/src/System/AssertExtensions.cs
```

Comment:
This is not that common to have it used in all assemblies

Reply:
Ok, thanks, I will reconsider it to make it more assembly-specific.

Reply:
addressed